README.md

```diff
@@ -75,15 +86,15 @@ const api = new eosjs.Api({ rpc, signatureProvider, textDecoder: new TextDecoder
 
 ### Error handling
 
-use `eosjs_jsonrpc.RpcError` for handling JSON-RPC Errors
+use `RpcError` for handling RPC Errors
```

Comment:
`Use` capitalized

Reply:
I think i preferred the JSON-RPC Errors  instead of just RPC from the older line, since it's the json rpc class

Reply:
We may implement other RPC formats in the future and the error will work for all formats, so I left this non-specific